stream/src/placeholder/handshake.rs

```diff
@@ -18,32 +21,35 @@ fn suffix_namespace(namespace: &[u8]) -> Vec<u8> {
     union(namespace, NAMESPACE_SUFFIX_HANDSHAKE)
 }
 
-pub fn create_handshake<E: Clock, C: Scheme>(
-    runtime: E,
+/// Get the current time in milliseconds since the Unix epoch.
+/// If the system clock is set to before the Unix epoch, this function will panic.
+/// The result saturates at the maximum value of u64.
+pub fn get_current_time_ms<E: Clock> (runtime: &E) -> u64 {
+    let time = runtime.current()
+        .duration_since(UNIX_EPOCH)
+        .expect("failed to get current time");
+    time::to_millis(time)
+}
+
+pub fn create_handshake<C: Scheme>(
     crypto: &mut C,
     namespace: &[u8],
+    timestamp_ms: u64,
     recipient_public_key: PublicKey,
     ephemeral_public_key: x25519_dalek::PublicKey,
 ) -> Result<Bytes, Error> {
```

Comment:
I changed `create_handshake` to take the timestamp explicitly (rather than taking in `runtime`) which makes testing easier IMO, let me know what you think

Reply:
I think that makes sense. Agree it should make testing easier.